docs/source/en/_toctree.yml

```diff
@@ -1,279 +1,303 @@
-- sections:
+- title: Get started
+  sections:
   - local: index
-    title: 🤗 Transformers
+    title: Transformers
   - local: quicktour
-    title: Quick tour
+    title: Quickstart
   - local: installation
     title: Installation
-  - local: add_new_model
-    title: Adding a new model to `transformers`
-  title: Get started
-- sections:
-  - local: pipeline_tutorial
-    title: Run inference with pipelines
-  - local: autoclass_tutorial
-    title: Write portable code with AutoClass
-  - local: preprocessing
-    title: Preprocess data
-  - local: training
-    title: Fine-tune a pretrained model
-  - local: run_scripts
-    title: Train with a script
-  - local: accelerate
-    title: Set up distributed training with 🤗 Accelerate
-  - local: peft
-    title: Load and train adapters with 🤗 PEFT
-  - local: model_sharing
-    title: Share your model
-  - local: agents
-    title: Agents 101
-  - local: agents_advanced
-    title: Agents, supercharged - Multi-agents, External tools, and more
-  - local: llm_tutorial
-    title: Generation with LLMs
-  - local: conversations
-    title: Chatting with Transformers
-  title: Tutorials
-- sections:
-  - isExpanded: false
-    sections:
-    - local: tasks/sequence_classification
-      title: Text classification
-    - local: tasks/token_classification
-      title: Token classification
-    - local: tasks/question_answering
-      title: Question answering
-    - local: tasks/language_modeling
-      title: Causal language modeling
-    - local: tasks/masked_language_modeling
-      title: Masked language modeling
-    - local: tasks/translation
-      title: Translation
-    - local: tasks/summarization
-      title: Summarization
-    - local: tasks/multiple_choice
-      title: Multiple choice
-    title: Natural Language Processing
-  - isExpanded: false
+- title: Base classes
+  isExpanded: False
+  sections:
+  - title: Models
     sections:
-    - local: tasks/audio_classification
-      title: Audio classification
-    - local: tasks/asr
-      title: Automatic speech recognition
-    title: Audio
-  - isExpanded: false
+    - local: models
+      title: Loading
+    - local: custom_models
+      title: Customizing models
+    - local: how_to_hack_models
+      title: Customizing model components
+    - local: model_sharing
+      title: Sharing
+    - local: add_new_model
+      title: Adding a new model
+    - local: modular_transformers
+      title: Modular Transformers
```

Comment:
This might be better under adding a model, or close to the philosophy

Reply:
I think since Adding a new model is already so long, its better to keep Modular Transformers separate. Having them next to each other is ideal so users don't have to skip around too much in the sidebar!